.env.template

```diff
@@ -39,28 +39,22 @@ REACT_APP_TEKU_INSTALLATION_URL=https://docs.teku.consensys.io/get-started/insta
 # true, false
 REACT_APP_IS_MAINNET=false
 # string (if `REACT_APP_IS_MAINNET=true`, it's the active public testnet name; otherwise, it's the testnet name of this website)
-REACT_APP_TESTNET_LAUNCHPAD_NAME=Holesky
+REACT_APP_TESTNET_LAUNCHPAD_NAME=Hoodi
 # string
-REACT_APP_CONTRACT_ADDRESS=0x4242424242424242424242424242424242424242
-# string
-REACT_APP_RPC_URL=https://rpc.holesky.ethpandaops.io
+REACT_APP_CONTRACT_ADDRESS=0x00000000219ab540356cBB839Cbe05303d7705Fa
 # string
-REACT_APP_BEACONCHAIN_URL=https://holesky.beaconcha.in
+REACT_APP_RPC_URL=https://rpc.hoodi.ethpandaops.io
 # string
-REACT_APP_EL_EXPLORER_URL=https://holesky.beaconcha.in
+REACT_APP_BEACONCHAIN_URL=https://hoodi.beaconcha.in
 # string
-REACT_APP_TUTORIAL_URL=https://notes.ethereum.org/@launchpad/holesky
+REACT_APP_EL_EXPLORER_URL=https://hoodi.beaconcha.in
 # number
 REACT_APP_ETH_DEPOSIT_OFFSET=0
 # string
-REACT_APP_FAUCET_URL=https://www.holeskyfaucet.io/
-# string
-REACT_APP_RPC_URL=https://rpc.holesky.ethpandaops.io
+REACT_APP_FAUCET_URL=https://hoodi-faucet.pk910.de/
 # string
-REACT_APP_BEACONCHAIN_URL=https://holesky.beaconcha.in
+REACT_APP_BEACONCHAIN_URL=https://dora.hoodi.ethpandaops.io/
```

Comment:
Is this used anywhere in terms of deployment? If so, we might wanna switch to using `https://hoodi.beaconcha.in` instead here. Even if we want to use Dora, we would need to remove the leading `/` to make it work.

Reply:
No, this is just the template for users to in their local environments... can patch now separately in a small PR but don't think this needs to block.

Reply:
Addressed in #757